composer.json

```diff
@@ -15,10 +15,13 @@
     },
     "config": {
         "optimize-autoloader": true,
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "bamarni/composer-bin-plugin": true
```

Comment:
@norberttech do you object to using this?

It allows to isolate your dependencies and not tie your project's dependencies to dependencies of CI tools (php-cs-fixer, phpstan & psalm).

https://github.com/bamarni/composer-bin-plugin

Reply:
interesting, so far I was just putting tools in another folder, [like here](https://github.com/flow-php/flow/tree/1.x/tools) but this might be even better. Lets try it!